src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/EditMailingInfoModal/EditMailingInfoModal.tsx

```diff
@@ -162,7 +162,15 @@ export const EditMailingInfoModal: React.FC<EditMailingInfoModalProps> = ({
                         )
                       }
                     >
-                      {Object.values(SendNewsletterEnum).map((value) => (
+                      {[
```

Comment:
Since I don't expect this enum to change often if ever, I think it would be simpler and clearer to just provide an array with the desired order and map over that array.

Reply:
All good! What you had would work great but is probably overkill in this particular case.